e2e-tests/cypress/e2e/journeys/insurance/your-buyer/your-buyer.spec.js

```diff
@@ -16,6 +16,8 @@ const {
   BUYER_COUNTRY_LABEL,
   BUYER_ORGANISATION,
   BUYER_ORGANISATION_LABEL,
+  BUYER_ADDRESS,
+  BUYER_ADDRESS_LABEL,
```

Comment:
The destructuring is labelled as `YOUR_BUYER_FIELD_IDS` but it's importing `FIELDS`, need to rename